modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/AbstractResourceManager.java

```diff
@@ -64,6 +63,26 @@ public static JSONDecoder getDecoder() throws CharonException {
         return decoder;
     }
 
+    /**
+     * Returns the resource URL builder implementation.
+     *
+     * @return AbstractResourceURLBuilder.
+     */
+    public static AbstractResourceURLBuilder getResourceURLBuilder() {
```

Comment:
I don't think this getter is needed.
This is set to allow url building options further